src/__tests__/components/loader.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BrowserRouter } from 'react-router-dom';
+import Loader from '../../components/loader/Index';
+
+describe('Loader component', () => {
```

Comment:
@leksyib I thought this was tested in a previous PR?

Reply:
@9jaswag  I made this branch from the previous one

Reply:
That's why it has commits from the previously merged branch